made a mobile, tablet and desktop version

Alex0041 opened the pull request:
сделал мобильную, планшетную и десктопную версию.
ЕЩЕ НЕ ВСЕ СДЕЛАНО.
Еще нужно (ховеры/фокусы, свг на кнопки).
НО ОТПРАВЛЯЮ, ЧТОБЫ УВИДЕЛИ КОНФЛИКТ МОЕЙ СЕКЦИИ И ТАНИНОЙ. ЕЙ НУЖНО УВЕЛИЧИТЬ СВОИ СЕКЦИИ

Alex0041 committed:
made a mobile, tablet and desktop version

сделал мобильную, планшетную и десктопную версию.
ЕЩЕ НЕ ВСЕ СДЕЛАНО.
Еще нужно (ховеры/фокусы, свг на кнопки).
НО ОТПРАВЛЯЮ, ЧТОБЫ УВИДЕЛИ КОНФЛИКТ МОЕЙ СЕКЦИИ И ТАНИНОЙ. ЕЙ НУЖНО УВЕЛИЧИТЬ СВОИ СЕКЦИИ

Oksana1594 commented on src/sass/_contacts.scss:
```diff
@@ -0,0 +1,260 @@
+.contacts-section {
+  position: relative;
+  background-color: #84a178;
+  padding-bottom: 40px;
```
для цієї секції різні нижні падінги на всіх пристроях (м-32, т-40, д-120)

Alex0041 replied:
спасибо, исправил

Oksana1594 commented on src/sass/_contacts.scss:
```diff
@@ -0,0 +1,260 @@
+  text-align: center;
+}
+
+.contacts__category {
```
тут колір змінюється на темний на таблеті та десктопі

Alex0041 replied:
исправил

Oksana1594 commented on src/sass/_contacts.scss:
```diff
@@ -0,0 +1,260 @@
+}
+
+.contacts__category {
+  font-family: 'Montserrat';
```
фонто фемелі Монсеррат задавала на баді. має наслідуватись, приберіть по стилям де не потрібно

Alex0041 replied:
убрал

Oksana1594 commented on src/sass/_contacts.scss:
```diff
@@ -0,0 +1,260 @@
+  display: block;
+  padding: 12px 34px;
+  align-self: center;
+  margin-top: 16px;
```
марджин топ на телефоні має бути 8 пкс

Oksana1594 replied:
падінги на попередню секцію додані, все відображається ок

Alex0041 replied:
тут не зовсім зрозумів, де саме?
між кнопками? то там 16px

Alex0041 committed:
corrected remarks

исправил замечания, пофиксил что заметил и добавил ховер/фокус на кнопки.
Осталось сделать ховер/фокус на mail/tel и добавить svg на кнопки
